components/LibraryIndex/LibraryIndex.jsx

```diff
@@ -1,5 +1,24 @@
-function LibraryIndex() {
-	return <></>
+function LibraryIndex(props) {
+	const {libraryItems} = props
+
+	const allLibraryItems = libraryItems.map((libraryItem, index) => (
+		<ul key={index}>
+			<li>Name: {libraryItem.name}</li>
```

Comment:
This should be a link that naviagets to an individual item.

Reply:
These should now link to each individual library item